base_domain_operator/expression.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api
+# pylint: disable=W0402
+from openerp.osv.expression import ExtendedLeaf, expression, is_operator,\
```

Comment:
FYI It was fixed from https://github.com/OCA/pylint-odoo/pull/118
Wa didn't have `from openerp.osv.expression...` import case supported (Sorry)